_assets/css/styles.scss

```diff
@@ -10,25 +10,25 @@
 
 // -------------------------------------
 // Import individual theme settings
-@import 'theme/uswds-theme-general';
```

Comment:
not sure what lint you're using, but might be nice to check in the .eslintconfig eventually

Reply:
This is from `prettier`, the new `site` repo uses it and has the config checked in.